packages/manager/modules/exchange/src/account/update/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,78 @@
+{
+  "exchange_account_update_back_button": "Modifier le compte Exchange",
+  "exchange_account_update_header": "Vous pouvez ajouter un compte e-mail à votre offre. Il sera pris en compte dans votre facture mensuelle.",
+  "exchange_account_update_personal_account_info_title": "Informations du compte",
+  "exchange_account_update_company_account_info_title": "Informations de la sociéte",
+  "exchange_account_update_contact_account_info_title": "Informations de contact",
+  "exchange_account_update_sections_title": "Paramètres avancés",
+  "exchange_account_update_accountType_label": "Type de compte (obligatoire)",
+  "exchange_account_update_accountType_tooltip": "E-mail pro : profitez de 10 Go de stockage, des protocoles POP, IMAP et SMTP.\nExchange : profitez de 50 Go de stockage, des protocoles MAPI et ActiveSync.\n",
+  "exchange_account_update_emailAddress_label": "Adresse e-mail (obligatoire)",
+  "exchange_account_update_emailAddress_hint": "La partie locale de votre adresse e-mail (le texte situé avant le « @ ») doit respecter ces indications :<ul>\n<li qtlid=\"485957\">elle doit débuter et se terminer par une lettre ou un chiffre ;</li>\n<li qtlid=\"485970\">les caractères spéciaux autorisés sont « . », « + », « - » et « _ » ;</li>\n<li qtlid=\"485983\">il est impossible de mettre côte à côte des caractères spéciaux.</li>\n</ul>",
```

Comment:
**nitpick:** Can this translation be cleaned up a bit? You can safely remove `qtlid=""` attributes.

Reply:
This is coming from my refactorig task. It was there before.
Just a copy-past.

Reply:
Nice to have it before starting the translation process.
We are referring to static content here and it could be applied easily.

**Translation before:**

```json
{
  "exchange_account_update_emailAddress_hint": "La partie locale de votre adresse e-mail (le texte situé avant le « @ ») doit respecter ces indications :<ul>\n<li qtlid=\"485957\">elle doit débuter et se terminer par une lettre ou un chiffre ;</li>\n<li qtlid=\"485970\">les caractères spéciaux autorisés sont « . », « + », « - » et « _ » ;</li>\n<li qtlid=\"485983\">il est impossible de mettre côte à côte des caractères spéciaux.</li>\n</ul>",
}
```

**Translation after:**
```json
{
  "exchange_account_update_emailAddress_hint": "La partie locale de votre adresse e-mail (le texte situé avant le « @ ») doit respecter ces indications :",
  "exchange_account_update_emailAddress_hint_info_1": "elle doit débuter et se terminer par une lettre ou un chiffre ;",
  "exchange_account_update_emailAddress_hint_info_2": "les caractères spéciaux autorisés sont « . », « + », « - » et « _ » ;",
  "exchange_account_update_emailAddress_hint_info_3": "il est impossible de mettre côte à côte des caractères spéciaux.",
}
```

**Template before:**

```html
<div data-ng-bind-html="'exchange_account_add_emailAddress_hint' | translate"></div>
```

**Template after:**

```html
<div>
  <span data-translate="'exchange_account_update_emailAddress_hint'"></span>
  <ul>
    <li data-translate="exchange_account_update_emailAddress_hint_info_1"></li>
    <li data-translate="exchange_account_update_emailAddress_hint_info_2"></li>
    <li data-translate="exchange_account_update_emailAddress_hint_info_3"></li>
  </ul>
</div>
```

Reply:
Thanks @antleblanc, realy i can't.
I have others task to complete and i already spent lot time on that one.

We have to plan anothe refactoring task.

Reply:
Here we are adding new translations that are required but we have to take into in account the translation process.

Submitting such translations doesn't make sense. 
```txt
<li qtlid=\"485957\">...
```

Its bring unnecessary complexity to the translation itself (complex to maintain).
It won't help the translation team because they'll have to keep some escaped characters and unnecessary attributes like `qdtlid`.

Reply:
Done